packages/core/src/App/Components/Layout/Header/menu-links.jsx

```diff
@@ -35,7 +34,11 @@ const MenuLinks = ({ is_logged_in, items, is_pre_appstore }) => (
                 {items.map(item => {
                     return (
                         is_logged_in && (
-                            <MenuItems item={item} hide_menu_item={item.text() === 'Reports' && is_pre_appstore} />
+                            <MenuItems
+                                item={item}
+                                hide_menu_item={item.text() === 'Reports' && is_pre_appstore}
+                                key={`${item.icon}${item.id}`}
```

Comment:
```suggestion
                                key={`${item.icon}${item.id}`}
```
Nit: move the key as the first prop